src/stories/css/input.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+    font-family: 'VisbyCF-ExtraBold';
+    src: url('../assets/Fonts/VisbyCF/VisbyCF-ExtraBold.ttf')  format('truetype');
+  }
+@font-face {
+font-family: 'OpenSans';
+src: url('../assets/Fonts/Open_Sans/OpenSans-Regular.ttf')  format('truetype');
+}
+input{
+    font-family: 'OpenSans', sans-serif;
+    font-weight: 400;
+    outline: none;
+    border: none;
+}
+
+.input-icon--left{
+    width: 20px;
+    height: auto;
+    padding-right: 9px;
+}
+.input-icon--right{
+    width: 20px;
+    height: auto;
+    padding-right: 9px;
+}
+.storybook-input-div {
+    font-family: 'OpenSans', sans-serif;
+    display: flex;
+    flex-direction: column;
+    font-weight: 400;
+    width: fit-content;
+    flex-basis: auto;
+
+}
+.storybook-input-label {
+    padding-bottom: 8px;
+}
+.storybook-input {
+    font-family: 'OpenSans', sans-serif;
+    width: auto;
+    padding: 12px 16px 12px 16px;
+    border-radius: 4px;
+    border: 1px solid rgba(217, 217, 217, 1);
+    display: flex;
+    flex-direction: row;
+}
+.storybook-input:hover {
+    border: 1px solid rgba(90, 80, 179, 1);
+}
+.storybook-input:focus {
+    outline: none;
+    border: 1px solid rgba(90, 80, 179, 1);
+}
+
+.storybook-input--disabled {
+    border: 1px solid rgba(217, 217, 217, 1);
+    color: #D9D9D9;
+    background-color: #FAFBFC;
+}
+
+.storybook-input--disabled div:hover {
+    border: 1px solid rgba(217, 217, 217, 1);
```

Comment:
There shouldn't be a change in the input component on hover when it is disabled. Furthermore, this CSS code doesn't seem to have an impact on the disabled input field. (most likely being overwritten)

Reply:
Yeah... I was trying to get it so that when it is disabled hovering won't change the border color. It was not a problem when the react input component was our input component, but when I added the div to add the icons is when it stopped working as intended.

Reply:
Also to add to that, when the input is disabled, I haven't figured out how to get the div also to be the same color at the disabled internal input. (You can see what I mean in the video I added to the pr)